kahuna/public/js/components/gr-top-bar/gr-top-bar.css

```diff
@@ -13,15 +13,19 @@ gr-top-bar-nav {
 
 gr-top-bar-actions {
     display: flex;
+    min-width: 100px;
+    flex-direction: row;
+    justify-content: flex-end;
 }
 
 
 .gr-top-bar-inner {
     background-color: #333;
-    padding: 10px;
+    padding: 0px 10px 0px 0px;
```

Comment:
Ah, I was using the Chrome console as an editor, so I guess this is an artefact of that.